app/assets/stylesheets/layout/page_specific/_money_navigator_questionnaire.scss

```diff
@@ -132,6 +132,10 @@
 			padding: 0 0 0 $baseline-unit * 4;
 		}
 	}
+
+	.question__response__hidden {
+		display: none;
+	}
```

Comment:
Could we move this rule to be within the `[data-dough-money-navigator-questions-initialised="yes"]` section of the CSS (below)? That way this will only apply to the enhanced display of the questionnaire. Probably right at the top would be fine since it is a useful general rule.